src/libaducpal/src/stdio.c

```diff
@@ -4,27 +4,26 @@
 
 #    include <direct.h> // _rmdir
 #    include <io.h> // _unlink
+#    include <stdio.h> // popen, pclose
 
 #    include <errno.h> // ENOTDIR, _get_errno
 
 FILE* ADUCPAL_popen(const char* command, const char* type)
 {
-    __debugbreak();
-    return NULL;
+    // TODO(JeffMill): [PAL] Will this return utf8? If not, additional work is needed.
+    return _popen(command, type);
```

Comment:
why is there underscore on front?  Also why not pclose(stream) instead of _pclose(stream) on line 19?

Reply:
underscore is the POSIX compatible name.